### ISSUE TITLE
asyncio timeout exception not caught

### ISSUE THREAD

**Issue by sjefferson99** [somakeit/smib#99]
The timeout exception introduced by the following line in hid.py is not caught by except Exception:
`new_space_state = await wait_for(self.slack_api.async_get_space_state(), self.checking_space_state_timeout_s)`

The coroutine is correctly stopped, but an error is posted to the log instead of the exception being caught in the try/except block.

```
Task exception wasn't retrieved
future: <Task> coro= <generator object 'async_update_space_state_output' at 20021080>
Traceback (most recent call last):
  File "asyncio/core.py", line 1, in run_until_complete
  File "lib/hid.py", line 141, in async_update_space_state_output
  File "asyncio/funcs.py", line 1, in wait_for
TimeoutError: 
```

**Comment by sjefferson99** [comment]
To reproduce, insert an await asyncio.sleep(35) in the slack_api.async_get_space_state() method to simulate a slow web response beyond the timeout.

**Comment by sjefferson99** [comment]
This was fixed in #111
